packages/client/hmi-client/src/workflow/ops/model-transformer/tera-model-transformer.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+	<div class="background">
+		<Suspense>
+			<tera-model-jupyter-panel
+				asset-id="bcd32c48-1daa-4a52-8ed7-147c1dc4b0b0"
+				:model-configuration-id="modelConfigurationId"
```

Comment:
Why do we have a hardwired `bcd32c48-1daa-4a52-8ed7-147c1dc4b0b0`

Reply:
sorry i believethe asset-id is not longer used and I was using it for testing

Reply:
this has been removed...